skype/skype-ps/skype/Set-CsTeamsGuestMeetingConfiguration.md

```diff
@@ -139,6 +139,17 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -LiveCaptionsEnabledType
+Determines whether real-time captions are available for the Guest users in Teams meetings. Set this to DisabledUserOverride to allow user to turn on live captions. Set this to Disabled to prohibit.
```

Comment:
Small style suggestions: looks like guest isn't capitalized in the rest of the file. Also seems like other descriptions just say "guests" instead of guest users.